ferveo-python/ferveo/__init__.pyi

```diff
@@ -173,15 +193,16 @@ class SharedSecret:
         ...
 
 
+@final
 class FerveoVariant:
-    @staticmethod
-    def simple() -> str: ...
+    simple: FerveoVariant
```

Comment:
One minor nitpick I didn't pick up on earlier (and I know it's annoying), but can these be `Simple` and `Precomputed`. The lowercase looks a little weird in the Python code 😅

Reply:
Ah I see. Admittedly, it's a nitpick so if it is easy to fix/makes sense then great; if not, then you can ignore.

I didn't notice it until I saw it used in the 3171.